man/mono.1

```diff
@@ -1809,6 +1809,10 @@ Enables the maximum JIT verbosity for the specified method. This is
 very helpfull to diagnose a miscompilation problems of a specific
 method.
 .TP
+\fBMONO_CFG_DUMP_METHOD\fR
```

Comment:
Can we call this e.g. `MONO_JIT_DUMP_METHOD` or something similar instead?
We already have the totally unrelated (albeit rarely used) `MONO_CFG_DIR` and I'd like to avoid confusion.